### PR TITLE
Allow using non-default VPCs

### PR DESCRIPTION
**Pull request by kmoppel**
A new manifest attribute / CLI flag --aws-vpc-id + README update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,69 +205,56 @@ docker run --rm --name pg1 -e PGSO_INSTANCE_NAME=pg1 -e PGSO_REGION=eu-north-1 \
...
-docker run --rm --name pg1 -e PGSO_TEARDOWN_REGION=y -e PGSO_REGION=eu-north-1 \
-  -e PGSO_AWS_ACCESS_KEY_ID="$(grep -m1 aws_access_key_id ~/.aws/credentials | sed 's/aws_access_key_id = //')" \
-  -e PGSO_AWS_SECRET_ACCESS_KEY="$(grep -m1 aws_secret_access_key ~/.aws/credentials | sed 's/aws_secret_access_key = //')" \
-  pgspotops/pg-spot-operator:latest
-```
+By default, security is not super trimmed down, as the experience is geared towards more casual or temporary workloads.
+Everything is tunable though.
 
-or by running a helper script from the "scripts" folders (by default just lists the resources, need to add a parameter):
+**ATTENTION** For all public Spot Operator managed instances it is very much **recommended to create a new special
```
Comment (scorpevans) [inline README.md:213]: what does public in `public Spot Operator managed instances` mean?

File: README.md [hunk 2]
```diff
@@ -205,69 +205,56 @@ docker run --rm --name pg1 -e PGSO_INSTANCE_NAME=pg1 -e PGSO_REGION=eu-north-1 \
...
 
-PS The operator tags all created object with special `pg-spot-operator-\*` tags, thus uses those also for the cleanup.
+## Security-relevant attributes with defaults
 
-# Local non-cloud development
+When using Docker / CLI:
 
-For Postgres setup testing, manifest handling etc, one can get by with local Virtualbox/Vagrant VMs, by using
-according flags:
+* --public-ip (Defaults to true)
```
Comment (scorpevans) [inline README.md:247]: we should really be using prefixes like `is/has/creates/uses/etc` e.g.

Comment (kmoppel) [reply README.md]: I've went with `--assign-public-ip` which is a bit better. "is/has" doesn't suit too well in this context I think. AWS for example uses `--associate-public-ip-address`

File: README.md (hunk 1 above)
Comment (kmoppel) [reply README.md]: Added that its about a publice IP

**Pull request merged**